project_tracker/config.py

```diff
@@ -1,3 +1,3 @@
-min_cluster_size = 10
-max_cluster_size = 600
-cluster_tolerance = 0.4
\ No newline at end of file
+min_cluster_size = 5
+max_cluster_size = 10000
```

Comment:
@amir-kt do you know why we limit max cluster size? Does it get slower with a bigger size?

Reply:
My intuition says that yeah it would make it slower and maybe it would even make it a bit confused in very dense areas. The reason we have those min and max values is that the library we're using requires those values. 
The extent of the explanation they gave in the library's docs in this:

> We impose that the clusters found must have at least setMinClusterSize() points and maximum setMaxClusterSize() points.